site/src/pages/WorkspacesPage/data.ts

```diff
@@ -0,0 +1,113 @@
+import {
```

Comment:
I might call this file `hooks.ts`

Reply:
Yes, I thought the same and I was using it but when thinking about what this does, it just loads/handles the page data so I thought it would be a better name. Makes sense?